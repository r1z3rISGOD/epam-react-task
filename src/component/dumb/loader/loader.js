"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Loader = void 0;
var React = require("react");
require("./loader.scss");
var Loader = function () {
    return (React.createElement("div", { className: "cssload-fond" },
        React.createElement("div", { className: "cssload-container-general" },
            React.createElement("div", { className: "cssload-internal" },
                React.createElement("div", { className: "cssload-ballcolor cssload-ball_1" })),
            React.createElement("div", { className: "cssload-internal" },
                React.createElement("div", { className: "cssload-ballcolor cssload-ball_2" })),
            React.createElement("div", { className: "cssload-internal" },
                React.createElement("div", { className: "cssload-ballcolor cssload-ball_3" })),
            React.createElement("div", { className: "cssload-internal" },
                React.createElement("div", { className: "cssload-ballcolor cssload-ball_4" })))));
};
exports.Loader = Loader;
//# sourceMappingURL=loader.js.map