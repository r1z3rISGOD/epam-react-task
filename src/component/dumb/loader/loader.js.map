{"version":3,"file":"loader.js","sourceRoot":"","sources":["loader.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,yBAAsB;AAEf,IAAM,MAAM,GAAc;IAC/B,OAAO,CACH,6BAAK,SAAS,EAAC,cAAc;QACzB,6BAAK,SAAS,EAAC,2BAA2B;YACtC,6BAAK,SAAS,EAAC,kBAAkB;gBAC7B,6BAAK,SAAS,EAAC,kCAAkC,GAC3C,CACJ;YACN,6BAAK,SAAS,EAAC,kBAAkB;gBAC7B,6BAAK,SAAS,EAAC,kCAAkC,GAC3C,CACJ;YACN,6BAAK,SAAS,EAAC,kBAAkB;gBAC7B,6BAAK,SAAS,EAAC,kCAAkC,GAC3C,CACJ;YACN,6BAAK,SAAS,EAAC,kBAAkB;gBAC7B,6BAAK,SAAS,EAAC,kCAAkC,GAC3C,CACJ,CACJ,CACJ,CACT,CAAA;AACH,CAAC,CAAA;AAvBY,QAAA,MAAM,UAuBlB"}