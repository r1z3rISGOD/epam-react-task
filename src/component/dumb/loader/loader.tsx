import * as React from 'react'
import './loader.scss'

export const Loader : React.FC = () => {
  return (
      <div className="cssload-fond">
          <div className="cssload-container-general">
              <div className="cssload-internal">
                  <div className="cssload-ballcolor cssload-ball_1">
                  </div>
              </div>
              <div className="cssload-internal">
                  <div className="cssload-ballcolor cssload-ball_2">
                  </div>
              </div>
              <div className="cssload-internal">
                  <div className="cssload-ballcolor cssload-ball_3">
                  </div>
              </div>
              <div className="cssload-internal">
                  <div className="cssload-ballcolor cssload-ball_4">
                  </div>
              </div>
          </div>
      </div>
  )
}
