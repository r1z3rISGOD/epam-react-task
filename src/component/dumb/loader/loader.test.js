import { Loader } from './loader.tsx'
import React from 'react'

it('loader', () => {
  const comp = shallow(<Loader/>)
  expect(comp).toMatchSnapshot()
})
