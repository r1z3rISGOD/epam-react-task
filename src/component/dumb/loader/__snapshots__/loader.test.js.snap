// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader 1`] = `
<div
  className="cssload-fond"
>
  <div
    className="cssload-container-general"
  >
    <div
      className="cssload-internal"
    >
      <div
        className="cssload-ballcolor cssload-ball_1"
      />
    </div>
    <div
      className="cssload-internal"
    >
      <div
        className="cssload-ballcolor cssload-ball_2"
      />
    </div>
    <div
      className="cssload-internal"
    >
      <div
        className="cssload-ballcolor cssload-ball_3"
      />
    </div>
    <div
      className="cssload-internal"
    >
      <div
        className="cssload-ballcolor cssload-ball_4"
      />
    </div>
  </div>
</div>
`;
